test/components/structures/auth/Login-test.tsx

```diff
@@ -319,4 +340,113 @@ describe("Login", function () {
         // error cleared
         expect(screen.queryByText("Your test-brand is misconfigured")).not.toBeInTheDocument();
     });
+
+    describe("OIDC native flow", () => {
+        const hsUrl = "https://matrix.org";
+        const isUrl = "https://vector.im";
+        const issuer = "https://test.com/";
+        const delegatedAuth = {
+            issuer,
+            registrationEndpoint: issuer + "register",
+            tokenEndpoint: issuer + "token",
+            authorizationEndpoint: issuer + "authorization",
+        };
+        beforeEach(() => {
+            jest.spyOn(logger, "error").mockClear();
```

Comment:
why do we need to `mockClear` here, *before* the test runs? If spies from previous tests can leak out, doesn't that mean that they can also leak to completely unrelated tests?